src/css/layout/helpers.less

```diff
@@ -10,9 +10,24 @@
   display: flex;
   align-items: center;
   justify-content: center;
-  height: 85vh;
-  width: 100vw;
+  height: 50vh;
   text-align: center;
+
+  .inline-dialog {
```

Comment:
I am sorry that I notice this so late.
But helpers.less isn't the right place for this.
This should go into an own file.
```inline-dialog.less```